File: ios/App/App/MyPlugin.m
//
//  MyPlugin.m
//  App
//
//  Created by Ronak on 19/10/20.
//

#import <Foundation/Foundation.h>
#import <Capacitor/Capacitor.h>

CAP_PLUGIN(CustomNativePlugin, "CustomNativePlugin",
           
           CAP_PLUGIN_METHOD(goToAppSetting, CAPPluginReturnPromise);
           )

File: ios/App/App/MyPlugin.swift
import Capacitor

@objc(NativePlugin)
public class CustomNativePlugin: CAPPlugin {
  @objc func goToAppSetting(_ call: CAPPluginCall) {
    let value = call.getString("value") ?? ""
    call.success([
        "value": value
    ])
  }
}